recipes/gottcha/meta.yaml

```diff
@@ -0,0 +1,32 @@
+package:
+  name: gottcha
+  version: "1.0"
+
+build:
+  number: 0
+  skip: False
+
+source:
+  fn: LANL-Bioinformatics-GOTTCHA-23a7f0d.tar.gz
+  url: https://github.com/LANL-Bioinformatics/GOTTCHA/tarball/master
```

Comment:
It would better to not use master as it will change. 

I would suggest to go for a specific commit, such as 
 `https://github.com/LANL-Bioinformatics/GOTTCHA/archive/23a7f0df7b5203a56354ed4926d40c19e7e8f6bd.tar.gz`